src/Okta.Sdk.IntegrationTests/UserTypesClientShould.cs

```diff
@@ -168,7 +168,6 @@ public async Task ReplaceUserType()
             }
         }
```

Comment:
We should rename this file to follow the "scenarios" pattern. You can do this separately if you want.